src/policy/fees.h

```diff
@@ -25,6 +25,9 @@
 // How often to flush fee estimates to fee_estimates.dat.
 static constexpr std::chrono::hours FEE_FLUSH_INTERVAL{1};
 
+// Stale fee estimates are not going to be read on mainnet
+static constexpr bool ACCEPT_STALE_FEE_ESTIMATES = false;
```

Comment:
in 2fec7f362d8540cda5ff7e3e4dfb2cb751364e06:

prefix with `DEFAULT_`, make doxygen compatible, use braced initialization. Documentation about where the option can be used belongs where that code lives, not here.

```suggestion
/** Whether we allow importing a fee_estimates file older than MAX_FILE_AGE. */
static constexpr bool DEFAULT_ACCEPT_STALE_FEE_ESTIMATES{false};
```

Reply:
Thank you, I updated it.